Lib/test/test_importlib/test_main.py

```diff
@@ -32,7 +32,7 @@ def test_new_style_classes(self):
 class ImportTests(fixtures.DistInfoPkg, unittest.TestCase):
     def test_import_nonexistent_module(self):
         # Ensure that the MetadataPathFinder does not crash an import of a
-        # non-existant module.
+        # non-existent module.
```

Comment:
```suggestion
        # nonexistent module.
```

Reply:
Accept this. I looked up 'non-existent versus nonexistent' and the hyphen used to be a British spelling, but is not now, and we use American spellings.

Reply:
Agreed that "nonexistent" is better.